### ISSUE TITLE
Priority can't be set via client

### ISSUE THREAD

**Issue by baldapps** [google/gcm#148]
It's not possible to set a priority for a message when you call send() method on Android. This is required in order to send a priority message to a device group where there isn't a server.

**Comment by kroikie** [comment]
Currently you cannot set the priority of a message from the client. I will make an internal feature request with the GCM team about adding this ability if it is feasible. I will update you on the feature request here.

A workaround would require a server which could relay the message to the device group and set the message priority.

**Comment by baldapps** [comment]
Yep, I know. However if Google gives possibility to use GCM without server, I think this feature is mandatory in order to use correctly this pattern. Thanks.

**Comment by baldapps** [comment]
Any news? Is it feasible?

**Comment by ogeidix** [comment]
No news on this for now, sorry.
But the feature request is tracked internally and we still hope to implement it in the future.
